Tasks/AzureCLIV1/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -16,7 +16,7 @@
   "loc.input.label.args": "Argumente",
   "loc.input.help.args": "An das Skript übergebenen Argumente",
   "loc.input.label.addSpnToEnvironment": "Auf Dienstprinzipaldetails im Skript zugreifen",
-  "loc.input.help.addSpnToEnvironment": "Hiermit werden Dienstprinzipal-ID und -schlüssel des ausgewählten Azure-Endpunkts zur Ausführungsumgebung des Skripts hinzugefügt. Sie können diese Variablen in Ihrem Skript verwenden: \"$servicePrincipalId\" und \"$servicePrincipalKey\".\n\nEine Berücksichtigung erfolgt nur, wenn der Azure-Endpunkt das Dienstprinzipal-Authentifizierungsschema aufweist.",
+  "loc.input.help.addSpnToEnvironment": "Hiermit werden Dienstprinzipal-ID und -schlüssel des ausgewählten Azure-Endpunkts zur Ausführungsumgebung des Skripts hinzugefügt. Sie können diese Variablen in Ihrem Skript verwenden: \"$servicePrincipalId\", \"$servicePrincipalKey\" und \"tenandId\"  .\n\nEine Berücksichtigung erfolgt nur, wenn der Azure-Endpunkt das Dienstprinzipal-Authentifizierungsschema aufweist.",
```

Comment:
Thanks for making changes in all these languages. But this is not strictly required - if you are not comfortable with the language and the translation. We translate the strings before every server release.

Reply:
It was a simple fix so i just did it.